13-gis.Rmd

```diff
@@ -133,28 +133,148 @@ In this case the output tells us, had we left the `FIELDS`-parameter unspecified
+- Leaving the output parameter(s) unspecified, saves the resulting QGIS output to a temporary folder created by QGIS.
+`run_qgis` prints these paths to the console after sucessfully runing the QGIS engine.
 - If the output consists of multiple files and you have set `load_output` to `TRUE`, `run_qgis` will return a list with each element corresponding to one output file.
 
 To learn more about **RQGIS** please refer to the (hopefully) forthcoming paper (cite). 
 
 ## (R)SAGA
-"SAGA CMD is a command line interface to execute SAGA modules.
-Other than SAGA GUI, it provides no functionality for data management and data visualization.
-It's purpose is to automate tasks by writing scripts or to execute SAGA modules from within an other application like a web server. 
-This section will describe the necessary setup of your system to use SAGA CMD and execute modules" [https://sourceforge.net/p/saga-gis/wiki/Executing%20Modules%20with%20SAGA%20CMD/](https://sourceforge.net/p/saga-gis/wiki/Executing%20Modules%20with%20SAGA%20CMD/).
+Similar to QGIS, the System for Automated Geoscientific Analyses (SAGA;  \@ref(tab:gis_comp)) provides the possibility to run SAGA modules from Python (SAGA Python API).
+In addition, there is also a command line interface (saga_cmd.exe) to execute SAGA modules (see also [https://sourceforge.net/p/saga-gis/wiki/Executing%20Modules%20with%20SAGA%20CMD/](https://sourceforge.net/p/saga-gis/wiki/Executing%20Modules%20with%20SAGA%20CMD/)).
+**RSAGA** uses the latter to run SAGA from within R.
+
+Though SAGA is a hybrid GIS, its main focus has been on raster processing, and here particularly on digital elevation models (soil properties, terrain attributes, climate parameters). 
+Hence, SAGA is especially good at the fast processing of large (high-resolution) rasters data sets [@conrad_system_2015]. 
+Therefore, we will introduce **RSAGA** with a raster and use case from @muenchow_geomorphic_2012.
+Specifically, we would like to compute the SAGA wetness index from a digital elevation model.
+First of all, we need to make sure that **RSAGA** will find SAGA on the computer when called.
+For this, all **RSAGA** functions using SAGA in the background make use of `rsaga.env()`. 
+Usually, `rsaga.env()` will detect SAGA automatically by searching several likely directories (see its help for more information). However, we have 'hidden' SAGA in the OSGEO4W-installation, a location `rsaga.env()` does not search automatically. Therefore, to ensure that `rsaga.env()` finds the SAGA installation, we add the corresponding path to the PATH environment variable. Naturally, these steps are unnecessary under Linux and MacOS. 
 
-hint to vignette("RSAGA-landslides", package = "RSAGA").
+```{r}
+library("RSAGA")
+path <- Sys.getenv("PATH")
+path <- paste(c("C:\\OSGeo4W64\\apps\\saga", path), collapse = ";")
```

Comment:
I have already updated this, see 06966e5. And since there is a tiny slash problem, I have opened a new link2GI issue (see https://github.com/gisma/link2GI/issues/3)